app/controllers/products_controller.rb

```diff
@@ -3,7 +3,7 @@ def index
   end
 
   def show
-    @product = Product.find_by_id(params[:id])
+    @product = Product.find_by_id(params[:id]).decorate
```

Comment:
**Nitpick - Code Quality**
You should make use of the `find_by` method like so: `Product.find_by(params[:id]).decorate`